docs/development.md

```diff
@@ -72,3 +72,29 @@ directory.
   - terminal.integrated.scrollback: you'll probably want more than 1000 lines of history
   - terminal.integrated.shell.windows: click the *Edit in settings.json* link and set the value
     `"terminal.integrated.shell.windows": "C:\\WINDOWS\\System32\\wsl.exe"`
+    
+### 5. Docker
+
+  - Install [Docker Desktop](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
+  - Go to Docker Settings on Windows and tick "Expose daemon on tcp://localhost:2375 without TLS" 
+  - These instructions are based off of [this guide - Docker and WSL](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly)
+  - Update package last  
+      `sudo apt-get update -y`
```

Comment:
Can you add the `add-apt-repository` step before this update?

Reply:
Will do